packages/typography/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@material/react-typography",
+  "version": "0.0.0",
+  "description": "Material Components React Typography",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "keywords": [
+    "mdc web react",
+    "material components react",
+    "material design",
+    "material typography",
+    "materialtypography"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web-react.git"
+  },
+  "dependencies": {
+    "@material/typography": "^0.40.1",
```

Comment:
lets aim to put this into https://github.com/material-components/material-components-web-react/tree/rc7.0. Please update to 0.41.0